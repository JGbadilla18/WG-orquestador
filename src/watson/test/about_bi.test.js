const testWatson = require('./testWatson');

watsonText = 'Acerca de BI.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
