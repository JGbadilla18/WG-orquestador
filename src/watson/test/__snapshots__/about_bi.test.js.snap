// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Acerca de BI. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Somos una sólida organización financiera, conscientes de nuestra responsabilidad con nuestros clientes y nuestro personal. Estamos #SiempreDeTuLado.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
